tools/mo/openvino/tools/mo/moc_frontend/pipeline.py

```diff
@@ -39,6 +39,20 @@ def check_places_are_same(places_original: List[Place], places_new: List[Place])
             [item for item in places_original if any(
                 [item.is_equal(item2['node']) for item2 in places_new])]) == len(places_original)
 
+    def add_names_to_tensors(model, places):
+        """
+        Adds additional names to some model input tensors. This helper should be used
+        when a model modification is going to happen.
+        :param model The input model loaded by a given frontend
+        :param places An object containing Places and names that will be used for model modification
+        """
+        for new_input in places:
+            try:
+                # some frontends might not implement this method
+                model.add_name_for_tensor(new_input['node'], new_input['input_name'])
+            except:
+                pass
```

Comment:
can we log warning to user's output that input with appropriate name will not be shown in IR in case of error?